assets/packages/questioneditor/src/components/subcomponents/_subquestions.vue

```diff
@@ -21,9 +21,11 @@
             >
                 <!-- Y-Scale -->
                 <div v-if="subquestionscale == 0" :key="subquestionscale" class="panel panel-default">
-                    <div class="panel-heading">
-                        {{'Y-Scale'|translate}}
-                    </div>
+                    <template v-if="subquestionScales.length > 1">
```

Comment:
The variable subquestioncale is just an array with max. 2 items. One Array for Y-Scale and one for X-Scale. I thin this is not needed cause. this part will only be rendered if subquestionscale is zero. otherwise it will not be shown. So the if is not needed in my opinion. Cause if subquestionScales.length is > than 1, it will render Y-Scale as text, but it will not be shown if subquestionscale is not 0. So this is unnecessary.

Reply:
Here : 

1. subquestionScales.length == 1 : didn't show it (array for example)
2. subquestionScales.length == 2 : show it (array text for example)

See the different picture at https://github.com/LimeSurvey/LimeSurvey/pull/1412#issuecomment-621112004

Reply:
If its 1 it will be not shown. But the general lenght of the subquestionScales  is 1.

Reply:
I want to show (default) Y-Scale only if X-Scale exist. Seems to work ?

Didn't show heading if scale <= 2 seems better ? Maybe not ?

Reply:
I mean : 

Current master (for Array (one scale, default) or multiple choice
![Capture d’écran du 2020-04-29 12-47-39](https://user-images.githubusercontent.com/1439428/80587913-ae37e100-8a17-11ea-8c60-936bbdad8538.png)


With this update : didn't show scale
![Capture d’écran du 2020-04-29 12-46-54](https://user-images.githubusercontent.com/1439428/80587952-bbed6680-8a17-11ea-8fba-9f8577c5b190.png)

Reply:
but leave the panel